spec/system/admin/users_spec.rb

```diff
@@ -90,6 +90,14 @@
           click_button "Clear key"
           expect(page).to have_content "NO KEY"
         end
+
+        it "should allow to disable the user" do
+          check "Disabled"
+          click_button "Update"
+
+          expect(page).to have_content("Account updated")
+          expect(page).to have_checked_field "Disabled"
+        end
```

Comment:
Just added the new `system` spec, tell me if any issue or adjustment.

Reply:
This is great. Can you add enabling as well? It can be in the same `it` block to save time running the specs.